server/src/main/java/org/apache/druid/server/initialization/ServerConfig.java

```diff
@@ -61,7 +63,9 @@ public ServerConfig(
       int inflateBufferSize,
       int compressionLevel,
       boolean enableForwardedRequestCustomizer,
-      @NotNull List<String> allowedHttpMethods
+      @NotNull List<String> allowedHttpMethods,
+      boolean sanitizeJettyErrorResponse,
```

Comment:
I think these configs should be called `enableErrorResponseTransform` and `errorResponseTransformStrategy` respectively, or something similar, since it is clearer their purpose (and also `ErrorResponseTransformStrategy` is the type name)

Reply:
sorry i got confused about  `sanitizeJettyErrorResponse`, it shouldn't be renamed to `enableErrorResponseTransform`, though `sanitizeDruidErrorResponse` should still be renamed `errorResponseTransformStrategy`

Reply:
We don't need `enableErrorResponseTransform` as not setting `errorResponseTransformStrategy` or setting `errorResponseTransformStrategy` to `none` would be the same as current behavior (no-op) and leaving the error response unchanged. 

I renamed `sanitizeJettyErrorResponse` to `showDetailedJettyErrors` and flip it 
also renamed `sanitizeDruidErrorResponse` to `errorResponseTransformStrategy`

Reply:
yep sorry confused myself, thinking about too many things at once 🙃 , this sgtm :+1:

Reply:
Actually it should be `errorResponseTransform` not `errorResponseTransformStrategy` for the @JsonProperty
so that the configs look something like
```
druid.server.http.showDetailedJettyErrors=false
druid.server.http.errorResponseTransform.strategy=allowedRegex
druid.server.http.errorResponseTransform.allowedRegex=["asd"]
```